### PR TITLE
Update chart-wrapper.component.ts

### PR DESCRIPTION
**Pull request by frsab**
fix https://github.com/FERNman/angular-google-charts/issues/173

### CODE REVIEW

**Review by FERNman — APPROVED** [review]

File: libs/angular-google-charts/src/lib/components/chart-wrapper/chart-wrapper.component.ts [hunk 1]
```diff
@@ -66,7 +66,7 @@ export class ChartWrapperComponent implements ChartBase, OnChanges, OnInit {
     return this.wrapper;
   }
 
-  public set chartWrapper(wrapper: google.visualization.ChartWrapper) {
+  public set chartWrapper(wrapper: google.visualization.ChartBase | undefined) {
```
Comment (FERNman) [inline libs/angular-google-charts/src/lib/components/chart-wrapper/chart-wrapper.component.ts:69]: This doesn't compile when using TS strict mode. Can you explain the reasoning behind this change?

**Review by FERNman — CHANGES_REQUESTED** [review]

**Comment by kussmaul** [comment]
I think we need to change the type for `set chartWrapper`, `get chartWrapper`, _and_ ChartBase.chartWrapper, e.g.:
https://github.com/kussmaul/angular-google-charts/commit/bf3a194e49a55fbbda2bef188e5bc02df44e89eb

**Comment by kussmaul** [comment]
Everything works for me with Angular-google-charts@2.0.1.
(My previous comment was for an earlier version. Sorry for the confusion.

**Comment by FERNman** [comment]
Closing as per @kussmaul's comment

**Pull request closed without merging**